publisher:
  ros__parameters:
   camera_url: "http://10.7.129.55:5000/video_feed"
   use_webcam: False
   webcam_device_id: 1